airflow/www/templates/admin/master.html

```diff
@@ -126,10 +127,81 @@
    }
    return readableFormat
 }
+
+// Capture keyboard shortcuts
+$(document).keyup(function(e) {
+  if (!/^(?:input|textarea|select|button)$/i.test(e.target.tagName)) {
+    let key = e.which || e.keyCode;
+    let letterT = 84;
```

Comment:
so its shift + t ?

Reply:
just the letter t

Reply:
shift + t  or T ?

Reply:
just `t`, no other modifiers :)

Reply:
do you think it should be something else? i was modeling this a little after github's `t` command

Reply:
hmm i think it can be two strokes because its easy to get the popup with just t and maybe thats not what you always want ?

Reply:
what is the case where you might accidentally get the popup that you're thinking about?

Reply:
When you are searching for a dag or filter it in the search textarea ?

I am going to assume that _!/^(?:input|textarea|select|button)$/i.test(e.target.tagName)_ if supposed to flag that and not show the popup ?

Reply:
yup, we exclude any typing from any elements that take user keyboard input (i didn't know buttons could but they apparently do according to SO 🤷 )

Reply:
i assume this is roughly how github does it too? https://docs.github.com/en/github/getting-started-with-github/keyboard-shortcuts#source-code-browsing

Reply:
going to merge and we can change to `[modifier] + t` if needed